frontend/src/components/ViewActivities/activitylist.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import app from '../Firebase';
+import * as activityFns from './activityfns.js';
+import ActivityDay from './activityday.js';
+import Accordion from 'react-bootstrap/Accordion';
+import '../../styles/activities.css';
+
+class ActivityList extends React.Component {
+  /** @inheritdoc */
+  constructor(props) {
+    super(props);
+    this.state = { days : [], tripId: props.tripId };
+  }
+
+  /** @inheritdoc */
+  async componentDidMount() {
+    if (this.state === null) { return; }
+    let tripActivities = await activityFns.getActivityList(this.state.tripId);
```

Comment:
Can this be null?

Reply:
Apparently, it can. And that makes problems.